desktop/apps/artwork/components/commercial/templates/inquire.jade

```diff
@@ -52,3 +52,17 @@ if artwork.is_inquireable
       data-context_type='sidebar'
     )
       | Contact #{artwork.partner.type}
+
+else if !artwork.is_for_sale && !artwork.is_acquireable
```

Comment:
I don't think this is quite right. The only thing we wanted to do was add the grey line on 'Not for Sale' works that were missing it.

There are valid reasons to not be showing the inquiry functionality on an artwork: namely, their partner may not be a partner that is 'allowed' to receive inquiries. All of that logic is contained in `is_inquireable`.